master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -261,7 +261,7 @@ private[celeborn] class Master(
           userResourceConsumption,
           requestId))
 
-    case requestSlots @ RequestSlots(_, _, _, _, _, _, _) =>
+    case requestSlots @ RequestSlots(_, _, _, _, _, _, _, _) =>
```

Comment:
I wonder if requests from old clients can be matched or not. Here needs some verification.

Reply:
> I wonder if requests from old clients can be matched or not. Here needs some verification.

Will default be false